galaxy_ng/app/api/ui/serializers/execution_environment.py

```diff
@@ -88,6 +94,9 @@ def get_updated(self, distro):
 
     def get_pulp(self, distro):
         repo = distro.repository
+        remote = None
+        if(repo.remote):
```

Comment:
unnecessary parenthesis 

```suggestion
        if repo.remote:
```

Reply:
woops. Javascript habits die hard.

Reply:
I'm surprised flake8 doesn't catch stuff like that. It cares about the most pedantic things like having a new line character at the end of each file.